client/webserver/site/src/js/app.js

```diff
@@ -105,6 +107,8 @@ export default class Application {
     // associated with  one of the available constructors.
     this.main = idel(document, 'main')
     const handler = this.main.dataset.handler
+    // Don't fetch the user until we know what page we're on.
+    await this.fetchUser()
```

Comment:
All seems to work well, although I want to confirm what's expected to happen here.  Before login or init, it is expected that a GET to /user will be a 401 unauthorized, correct?

Reply:
That is my intention.